server/src/Bible.controller.ts

```diff
@@ -57,6 +57,22 @@ export class BibleController {
         );
     }
 
+    @Get('/ref/:versionUid/:osisId/:chapterNr/:verseNr/plaintext')
+    async getVersePlaintext(
+        @Param('versionUid') versionUid: string,
+        @Param('osisId') osisId: string,
+        @Param('chapterNr') chapterNr: number,
+        @Param('verseNr') verseNr: number
+    ) {
+        const verseContent = await this.bibleEngine.getPlaintextForReferenceRange({
+            versionUid,
+            bookOsisId: osisId,
+            versionChapterNum: chapterNr,
+            versionVerseNum: verseNr
+        });
+        return { verseContent };
+    }
+
```

Comment:
maybe add

```
    @Post('/ref/plaintext')
    getReferenceRange(@Body() ref: IBibleReferenceRangeQuery) {
        return this.bibleEngine.getPlaintextForReferenceRange(ref);
    }
```

that way you can use it easily for e.g. crossrefs, which none of the other routes support (chapter spanning refs for example) and you don't need to add a plaintext version for every route

Reply:
one could also argue for using an optional query parameter `?plaintext=true`. this way we have one less method (and you would just set the `plaintext` parameter when calling the client method.